doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -373,15 +376,26 @@ When the proof is completed, you can exit proof mode with commands such as
            Proof. reflexivity. Qed.
 
            Lemma foo' : 0 = 0.
-           Proof using Type radixNotZero. reflexivity. Qed.
+           Proof using radixNotZero. reflexivity. Qed.  (* radixNotZero is not needed *)
 
       .. coqtop :: all
 
            Print foo'.   (* Doesn't show radixNotZero yet *)
          End bar.
          Print foo.      (* Doesn't change after the End *)
-         Print foo'.     (* End added type radix (used by radixNotZero) and radixNotZero *)
-
+         Print foo'.     (* "End" added type radix (used by radixNotZero) and radixNotZero *)
+         Goal 0 = 0.         
+         
+      .. coqtop :: in
+      
+         Fail apply foo'.  (* Fails because of the extra variable *)
+         
+      .. coqtop :: all
+      
+         apply (foo' 5).   (* Can be used if the extra variable is provided explicitly *)
```

Comment:
To be sure, `eapply foo’` should also work (and produce a new evar to be solved or goal to be proven).

Reply:
Yeah, but let's not get into that here.